source/edge-server/configure.txt

```diff
@@ -407,86 +278,140 @@ To stop the server, from the ``edge_server`` directory:
 Check the Edge Server Status
 ----------------------------
 
-When the Edge Server is running, you can see the status of the 
-Edge Server with:
+When the Edge Server instance is running, you can see the status of the 
+instance with:
 
 .. code-block:: shell
 
-   make status
+   edgectl status
 
 This gives you information about the number of clients connected to the 
 server, as well as errors that may have occurred.
 
-.. example::
-
-   The status object of an Edge Server may look similar to:
+The status object of an Edge Server instance may look similar to:
 
-   .. code-block:: json
-      :copyable: false
+.. code-block:: json
+   :copyable: false
 
-      {
-         "status": "ACTIVE",
-         "cloud_connected": true,
-         "num_local_clients": 1,
-         "query": {
-            "Item": "truepredicate"
-         }
+   {
```

Comment:
This has changed, I can provide an updated example in a few mins.

Reply:
Ah, good catch - thanks for mentioning this! I've got a working version locally so I can update this now. But I see there are now several new statuses in this object, with different messages - i.e.:

```
      "mongoContainerStatus": {
         "message": "running"
      },
      "mongoDBStatus": {
         "message": "ready"
      },
      "syncServerContainerStatus": {
         "message": "running"
      },
      "syncServerStatus": {
         "message": "ACTIVE"
      },
```

Just curious - is there anywhere I can see what the status message options are for each of these statuses, so I can document the options and what they mean?